tests/commands/test_runner.py

```diff
@@ -147,3 +149,23 @@ def test_patch_modules_from_env(self):
         update_patched_modules()
         assert EXTRA_PATCHED_MODULES["boto"] == True
         assert EXTRA_PATCHED_MODULES["django"] == False
+
+    def test_sitecustomize_run(self):
+        # [Regression test]: ensure users `sitecustomize.py` is properly loaded,
+        # so that our `bootstrap/sitecustomize.py` doesn't override the one
+        # defined in users' PYTHONPATH.
+        env = inject_sitecustomize('tests/commands/bootstrap')
+        out = subprocess.check_output(
+            ['ddtrace-run', 'python', 'tests/commands/ddtrace_run_sitecustomize.py'],
+            env=env,
+        )
+        assert out.startswith(b"Test success")
+
+    def test_sitecustomize_run_suppressed(self):
```

Comment:
@SeanOC added this test with `-S`. I've updated also the `ddtrace_run_sitecustomize.py` so that it checks if users `sitecustomize.py` has been loaded or not.